configure.ac

```diff
@@ -268,6 +269,12 @@ AC_HEADER_STDBOOL
 AC_C_CONST
 AC_TYPE_PID_T
 AC_HEADER_TIME
+AC_CHECK_TYPES([struct user_regs_struct, 
+                struct user_pt_regs,
+                struct user_fpregs_struct,
+                struct user_fpsimd_struct,
+                struct user_fpsimd_state]
```

Comment:
For some reason on our system this causes this definition in config.h:

#define HAVE_STRUCT_USER_FPSIMD_STATE_________________ 1

I found that if I move one of the commas to after this close bracket, then it defines it properly without all those trailing underscores.

Reply:
@agontarek: would it make sense to squash this commit to f09ed886cc2e9aa143f82453ca0b9c1c8962f11f?